packages/flutter_markdown/CHANGELOG.md

```diff
@@ -1,8 +1,12 @@
+## 0.6.10+2
+
+ * Migrates from `ui.hash*` to `Object.hash*`.
+
 ## 0.6.10+1
 
-* Updates Linux example to remove unneeded library dependencies that
+ * Updates Linux example to remove unneeded library dependencies that
   could cause build failures.
-* Updates for non-nullable bindings.
+ * Updates for non-nullable bindings.
```

Comment:
Rather than changing previous entries, just use `*` without a space before it going forward since that style has already been introduced. It's by far the most common across packages, and is what the new tooling for changelog management will do anyway.

Reply:
got it, reverted